app/services/auth/okta_service.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module Auth
+  class OktaService < BaseService
+    def authorize(email:)
```

Comment:
What about having something like

```ruby
module Auth
  module Okta
    class AuthorizeService
      def initialize(email:)
      ...

      def call
```

It will allow us to keep the `call` approach in the mutation: `Auth::Okta::AuthorizeService.call(email:)`

Reply:
I would need to refact `Auth::GoogleService` as well